docker_plugin/tasks.py

```diff
@@ -245,25 +243,23 @@ def pull(client, arguments):
 
     :node_property params: (Optional) Use any other parameter allowed
         by the docker API to Docker PY.
-    :param daemon_client: optional configuration for client creation
+    :param connection_credentials: optional configuration for client creation
     """
 
     arguments.update({'stream': True})
-    ctx.logger.info('Pull arguments: {0}'.format(arguments))
-
-    image_id = None
+    ctx.logger.info('Pull arguments: {0}'.format(str(arguments)))
 
     try:
         for stream in client.pull(**arguments):
             stream_dict = json.loads(stream)
-            if 'id' in stream_dict:
-                image_id = stream_dict.get('id')
-            if 'Complete' in stream_dict.get('status', ''):
-                ctx.logger.info('docker_plugin.tasks.pull: {0}.'.format(
-                    stream_dict))
+            if ('id' in stream_dict and
+                    'Complete' in stream_dict.get('status', '')):
+                ctx.logger.info(
+                    'docker_plugin.tasks.pull: {0}.'.format(
+                        stream_dict))
```

Comment:
Thanks, but please don't surround in parentheses as this makes it look like a tuple.